controllers/argocd/applicationset.go

```diff
@@ -44,8 +44,11 @@ func getArgoApplicationSetCommand(cr *argoproj.ArgoCD) []string {
 	cmd = append(cmd, "entrypoint.sh")
 	cmd = append(cmd, "argocd-applicationset-controller")
 
-	cmd = append(cmd, "--argocd-repo-server")
-	cmd = append(cmd, getRepoServerAddress(cr))
+	if cr.Spec.Repo.IsEnabled() {
+		cmd = append(cmd, "--argocd-repo-server", getRepoServerAddress(cr))
+	} else {
+		log.Info("Repo Server is disabled. This would affect the functioning of ApplicationSet Controller.")
```

Comment:
should this be a warning instead of Info?

Reply:
I didn't see a `log.Warn` option in the logger being used in the operator, thus ended up using `log.Info`.